cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/hbase/HBaseQueueAdmin.java

```diff
@@ -350,7 +351,8 @@ private void createConfigTable(String configTableName) throws IOException {
     htd.addFamily(hcd);
     hcd.setMaxVersions(1);
 
-    tableUtil.createTableIfNotExists(getHBaseAdmin(), tableName, htd, null,
+    // the TableId below is ignored, since the HTableDescriptor is created without it.
```

Comment:
not sure I understand.  So the table isn't created from the id we're passing in here?